app/client/src/components/ProgressBar.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div>
+    <b-progress :value="analysisProgress" :max="1" show-progress class="progress">
+        <b-progress-bar :value="analysisProgress" class="progress-bar" :animated="animated">
+        {{ (analysisProgress*100).toFixed(2) }}%
+        </b-progress-bar>
+    </b-progress>
+  </div>
+</template>
+
+<script lang="ts">
+import { mapGetters } from 'vuex';
+import { BProgress, BProgressBar } from 'bootstrap-vue-3';
+
+export default {
+  name: 'ProgressBar',
+  computed: {
+    ...mapGetters([
+      'analysisProgress',
+    ]),
+    animated() {
+      return this.analysisProgress as unknown as number !== 1;
```

Comment:
This double cast looks a little bit suspect!

Reply:
When removing the `as unknown` I get the following TS Error:
```
Conversion of type 'Computed' to type 'number' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
```
Seems that it cannot infer the return type from the mapped getter, even though I have specified it in getters.ts.
This happens also after using the GetterTree interface. Is it ok to just leave it like this, or should I rather try to import the getter in a different way that provides type information?

Reply:
OK, I think you might just need to declare your component using `defineComponent`, so 
```
import { defineComponent } from 'vue';

export default defineComponent({
  name: 'ProgressBar',
..etc...
 components: {
    BProgress,
    BProgressBar,
  },
});
```